containers/runtime/driver/Dockerfile

```diff
@@ -14,8 +14,29 @@
 
 FROM debian:buster
 
-RUN mkdir -p /src/workspace
+ARG REPOSITORY=grpc/grpc
+ARG GITREF=master
+
+RUN apt-get update && apt-get install -y git
+
+RUN mkdir -p /grpc/grpc
+WORKDIR /grpc/grpc
+
+RUN git clone https://github.com/$REPOSITORY.git .
+RUN git submodule update --init
+
+FROM l.gcr.io/google/bazel:0.17.1
+
+COPY --from=0 /grpc/grpc /src/workspace
+RUN mkdir -p /tmp/build_output
 WORKDIR /src/workspace
+RUN bazel --output_user_root=/tmp/build_output build //test/cpp/qps:qps_json_driver
+
+FROM debian:buster
```

Comment:
Any particular reasoning behind `debian:buster`?

Reply:
@jtattermusch said we should update from stretch to latest debian stable. I think that's buster. Is there another base image that would be good here?

Reply:
If you really wanted to be lean and mean, you could build on top of `scratch`. That's literally a blank filesystem. For (a ton) more effort, you could get a (somewhat) slimmer image. This is more difficult for languages like Python, which are Gordian knot of dependencies, but pretty manageable for always-statically-linked language like Go. Kubernetes-associated projects tend to use this approach since their images are used so frequently.

It's probably not worth implementing, but I figured it was worth mentioning.

Reply:
Good to note! Thanks 👍🏻